structuredevent-service-legacy/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/converter/StructuredEventToClusterShapeConverter.java

```diff
@@ -80,4 +90,28 @@
 
         return cdpClusterShape;
     }
+
+    private String getClusterTemplateDetails(String blueprintJson) {
```

Comment:
Please also cover this logic with UTs in StructuredEventToClusterShapeConverterTest